src/CarbonAware.Aggregators/src/CarbonAware/CarbonAwareAggregator.cs

```diff
@@ -127,15 +127,15 @@ private EmissionsForecast ProcessAndValidateForecast(EmissionsForecast forecast,
         forecast.Validate();
         forecast.ForecastData = IntervalHelper.FilterByDuration(forecast.ForecastData, forecast.DataStartAt, forecast.DataEndAt);
         forecast.ForecastData = forecast.ForecastData.RollingAverage(windowSize, forecast.DataStartAt, forecast.DataEndAt);
-        forecast.OptimalDataPoint = GetOptimalEmissions(forecast.ForecastData);
+        forecast.OptimalDataPoint = GetOptimalEmission(forecast.ForecastData);
```

Comment:
Yeah, I feel we should leave this for now.  It would be good to talk to the team that was using it in anger.  @bderusha - just thinking this should be `min = emissions.min` then `emissions.where( a => a.x == min)` to get all records where there are minimum values that match for forecast (as `minBy` only gets 1 record).  We can update but wanted to understand any downstream implications if we did this change.

Reply:
Since `EmissionsForecast` is an internal model it can be changed without compromising the WebAPI contract.  One would need to [update the method that converts the core model to the api DTO](https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/dev/src/CarbonAware.WebApi/src/Models/EmissionsForecastDTO.cs#L70)

I would also recommend changing the name within `EmissionsForecast` to `OptimalDataPoints` plural to better reflect the `IEnumerable` datatype.